src/Raven.Studio/wwwroot/App/views/database/indexes/indexes.html

```diff
@@ -18,7 +18,7 @@ <h2 class="text-center">No indexes have been created for this database.</h2>
                             <label></label>
                         </div>
                         <div class="btn-group-label" data-label="Filter">
-                            <input type="text" accesskey="/" class="form-control" placeholder="Index Name" title="Filter indexes (Alt+/)" data-bind="textInput: searchText" />
+                            <input type="text" accesskey="/" class="form-control" placeholder="Index Name" title="Filter indexes" data-bind="textInput: searchText" />
```

Comment:
hmm actually alt + / should work here -see accesskey="/". Could you check that?

Reply:
returning alt + / 
works :)